SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -2315,6 +2315,12 @@
                 "perftest"
             ]
         },
+        "OpenEuler": {
+            "license": "[BSD-3 License](https://github.com/pytorch/pytorch/blob/master/LICENSE)",
+            "specs": [
+                "python-pytorch"
+            ]
+        },
```

Comment:
I'm not too familiar with openEuler- I managed to find [their spec](https://gitee.com/src-openeuler/pytorch/blob/master/pytorch.spec), but I couldn't find any information on how they license the spec files they create. Mind pointing me to some documentation on the subject?

Reply:
After talking to CELA, we have decided to take the OpenEuler's PyTorch spec file under the PyTorch license (i.e., BSD-3), since there's no specific license found in OpenEuler's packages' repo.